HSStockChart/Classes/DrawLayerProtocol/HSDrawLayerProtocol.swift

```diff
@@ -225,4 +226,18 @@ extension HSDrawLayerProtocol {
         
         return CGSize(width: width, height: height)
     }
+	func getZeroCount(value:CGFloat) -> Int{
+		if value <= 0 {
+			return 0
+		}
+		let str = NSNumber.init(value: Double(value)).stringValue
```

Comment:
this conversion is too verbose，how about using `"\(value)"` of `value.description`